test/test_artifacts/v0/sm-python-sdk.test.Dockerfile

```diff
@@ -5,8 +5,7 @@ ARG MAMBA_DOCKERFILE_ACTIVATE=1
 RUN sudo apt-get update && sudo apt-get install -y git && \
     git clone --recursive https://github.com/aws/sagemaker-python-sdk.git && \
     :
-# Sagemaker Python SDK's unit tests requires AWS_DEFAULT_REGION to be set. So, using an arbitrary value of us-east-1
-ENV AWS_DEFAULT_REGION=us-east-1
```

Comment:
Why is this removed?

Reply:
It's designed to work in same way as the change I made above,
```

# Sagemaker Python SDK's unit tests requires AWS_DEFAULT_REGION to be set. So, using an arbitrary value of us-east-1
export AWS_DEFAULT_REGION=us-west-2
```
But it's not effective as tested locally. So removing it and count on commands in shell file directly.